bin/run_composable-kernels.sh

```diff
@@ -141,6 +172,11 @@ if [ "${ShouldInstallCK}" == 'yes' ]; then
   popd
 fi
 
+echo "Run suite: ${SelectedSuite}"
+
+# TODO: Correct indentation! For now: keep "wrong" one for easier review :)
```

Comment:
Just this TODO, I think we are good to go then

Reply:
Check and done :)